contributed_definitions/nyaml/NXfit_region.yaml

```diff
@@ -0,0 +1,92 @@
+category: base
+doc: |
+  Description of region for peak fitting (i.e., in an NXpeak_model).
+  
+  Used for descibring a subpart of the abscissa (in 2D spectra) that is
```

Comment:
What do you consider "2D" spectra here? I(x) or I(x, y)? Commonly, the dimension should be equal to the number of independent variables.
Whether to call the resulting graph a 2D or 1D plot is a different thing.

Reply:
removed